axlearn/common/test_utils.py

```diff
@@ -230,6 +238,16 @@ def assertNestedEqual(self, a, b):
             if hasattr(a_value, "dtype"):
                 self.assertEqual(a_value.dtype, b_value.dtype)
 
+    def assertAllClose(self, x, y, check_dtypes=True, rtol=1e-5, atol=1e-5, **kwargs):
```

Comment:
Do we need this, or can we use `assertNestedAllClose`?

Reply:
Remove this function. Considered done for this thread.